src/main/java/com/logbook/backend/logbookbe/domain/member/controller/MemberController.java

```diff
@@ -0,0 +1,138 @@
+package com.logbook.backend.logbookbe.domain.member.controller;
+
+import com.logbook.backend.logbookbe.domain.member.controller.dto.EditMemberResponse;
+import com.logbook.backend.logbookbe.domain.member.controller.dto.MemberResponse;
+import com.logbook.backend.logbookbe.domain.member.exception.MemberAlreadyExistsException;
+import com.logbook.backend.logbookbe.domain.member.exception.MemberUpdateException;
+import com.logbook.backend.logbookbe.domain.member.model.Member;
+import com.logbook.backend.logbookbe.domain.member.repository.MemberRepository;
+import com.logbook.backend.logbookbe.domain.member.service.MemberService;
+import com.logbook.backend.logbookbe.domain.project.model.Project;
+import com.logbook.backend.logbookbe.domain.project.repository.ProjectRepository;
+import com.logbook.backend.logbookbe.domain.user.model.User;
+import com.logbook.backend.logbookbe.domain.user.repository.UserRepository;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import com.logbook.backend.logbookbe.global.error.ErrorResponse;
+import com.logbook.backend.logbookbe.domain.member.exception.MemberNotFoundException;
+import com.logbook.backend.logbookbe.domain.user.exception.UserNotFoundException;
+import com.logbook.backend.logbookbe.domain.project.exception.ProjectNotFoundException;
+
+@RestController
+@RequestMapping("/api/members")
```

Comment:
요거 나중에
```suggestion
@RequestMapping("/api/projects/{project_id}/members")
```
로 바뀌나용??

중간발표 때 이렇게 바꾸기로 했었는데 명세서 수정을 안 했어요!! ㅜㅠ

Reply:
바꿨습니다~ 변경 사항 있으면 문서에 꼭 작성해주세요😅